### PR TITLE
Enriched the cheat sheet article with a little more text and descript…

### PR DESCRIPTION
**Pull request by SBortz**
…ion about how to use the patterns. Also added a Miro link for the cheat sheet in order to make it easy to copy & paste it into other miro boards.

### CODE REVIEW

File: data/authors.toml [hunk 1]
```diff
@@ -15,5 +15,5 @@ home_link = "https://www.linkedin.com/in/rafalmaciag/"
 
 [sbortz]
 display_name = "Sebastian Bortz"
-avatar = "https://pbs.twimg.com/profile_images/1196904725951795205/i9o8mYX0_400x400.jpg"
+avatar = "https://pbs.twimg.com/profile_images/1509963876229689357/ZUmVWQ79_400x400.jpg"
```
Comment (yordis) [inline data/authors.toml:18]: Do you mind downloading the image and adding it to https://github.com/event-modeling/eventmodeling.org/tree/master/static/avatars and using that instead?

Like https://github.com/event-modeling/eventmodeling.org/pull/26/files#diff-fb2c6dfdb52e5fe994810cb546767902efe4ef6e5765605df73eb9e90e462f4aR3

Comment (SBortz) [reply data/authors.toml]: Yep, good idea. 👍

**Comment by SBortz** [comment]
@adymitruk Hmm, i don´t know when i started using yellow as color for events. Somehow it felt natural for me.
But for the article, maybe i should change it to orange. Just in case anyone gets confused by the different colors.

**Comment by yordis** [comment]
@SBortz, I love your suggestion of switching to Orange, honestly, it felt weird to me since it has been a few years using Orange for that.

**Comment by SBortz** [comment]
@yordis: Ok, then i´ll update it.

**Comment by yordis** [comment]
Let me know once you have done it to merge it! 🚀

**Comment by SBortz** [comment]
Done :)

**Comment by SBortz** [comment]
It´s not online yet. Could anyone press the deployment button? :)

**Comment by yordis** [comment]
I can't help much on that front, sorry. @adymitruk can help!

**Pull request merged**